Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2077,6 +2077,7 @@ def get_remote_data_command(args: dict[str, Any]):
                       f'and last_update: {remote_args.last_update}')
         incident_type = find_incident_type(remote_incident_id)
         if incident_type == IncidentType.INCIDENT:
+            demisto.debug('incident found')
             mirrored_data, updated_object = get_remote_incident_data(remote_incident_id)
```

Comment:
lets log the data from this point

Reply:
it's logged in `set_xsoar_incident_entries` and there are no changes in between